faiss/impl/HNSW.h

```diff
@@ -142,9 +142,6 @@ struct HNSW {
     /// enough?
     bool check_relative_distance = true;
 
-    /// number of entry points in levels > 0.
-    int upper_beam = 1;
```

Comment:
Is it OK to just remove this and document in the next release? Is that typical for breaking changes thus far? FWIW I think I've seen some other folks setting beam in posted issues.